CHANGELOG.rst

```diff
@@ -2,6 +2,82 @@
 Changelog
 =========
 
+2.8.0-beta.0 (2024-09-17)
+=========================
+
+The (first) beta version for 2.8.0 is available for testing now.
+
+.. warning:: We encourage you to test out this beta version on non-production
+   environments and report your findings back to use. This release is not suitable for
+   production yet though.
+
+Upgrade notes
+-------------
+
+There are no manual actions required - all upgrades and migrations are automatic.
+
+.. note:: The UX rework in the ZGW APIs registration plugin is not entirely finished
+   yet. The Objects API integration in particular can be a bit confusing since it's not
+   possible yet to select which Objects API should be used. The plugin now uses the API
+   group that's listed first in the admin interface (**Admin** > **Miscellaneous** >
+   **Objects API Groups**).
+
+Detailed changes
+----------------
+
+**New features**
+
+* [:backend:`4577`] We improved the user experience when configuring the Objects API
+  registration plugin. Copy-pasting URLs is being phased out - you can now select the
+  relevant configurations in dropdowns.
+* [:backend:`4606`] Improved the user experience of the ZGW APIs registration plugin.
+  We're making this consistent with the Objects API. More improvements will be done in
+  the future.
+* [:backend:`4542`] Email components now support optional verification - when enabled,
+  users must verify their email address before they can continue submitting the form.
+* [:backend:`4582`] The SAML metadata for the DigiD/eHerkenning identity providers is
+  now automatically refreshed on a weekly basis.
+* [:backend:`4380`] The StUF-ZDS registration plugin now supports sending payment
+  details in the ``extraElementen`` data. For 2.7 this was available in an extension,
+  which has been merged in core - migrating is automatic.
+* [:backend:`4545`] You can now optionally configure an introduction page, which is
+  displayed before the start of the form.
+* [:backend:`4543`] You can now optionally enable a short progress summary showing the
+  current step number and the total number of steps in a form.
+
+**Unfinished features**
+
+.. note:: We've started preparing some improvements, but weren't able to fully finish
+   them yet. It's best not to use these in production yet.
+
+* [:backend:`4420`] We added support for additional validation in the postcode and city
+  fields of the ``addressNL`` component. However, it's not yet actived in the frontend.
```

Comment:
I rather not mention a feature that is not in, but do want to show the warning
and when and when not to use the addressNL component.

```suggestion
.. note:: The ``addressNL`` component is not yet a fully capable replacement for
   individual address fields. Currently, it's only recommended for 
   BRK-validation purposes.
```

Reply:
thanks, that's much better